browser/modules/Fathom.jsm

```diff
@@ -6,7 +6,7 @@
 
 var EXPORTED_SYMBOLS = ["Fathom"];
 
-var Fathom = {
+const Fathom = {
```

Comment:
Re: your commit message:
> Leave the `EXPORTED_SYMBOLS` symbol alone for now because no other modules dare to use anything but `var` in front of it, even if they use `const` just below. I wonder why.

`const` should be fine in a JSM; I had just copied it from another JSM that used `var`. JSMs should have their own scope that is not shared with anything else.

Reply:
Cool. I was being very cautious, but maybe I'll change it now for clarity.

Reply:
Heh, nm, all that code went away. :-)